scripts/py_bench.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# A sample script for running the Python Bench test
+
+IP="127.0.0.1"
+PORT="8889"
+
+#cp ./scripts/pythonContractConverter.py ./build/tools/bench/parsec/py/
+
+rr record ./build/tools/bench/parsec/py/py_bench --component_id=0 --ticket_machine0_endpoint=$IP:7777 --ticket_machine_count=1 --shard_count=1 --shard0_count=1 --shard00_endpoint=$IP:5556 --agent_count=1 --agent0_endpoint=$IP:$PORT
```

Comment:
I _love_ [`rr`](https://rr-project.org/), but we don't currently require it or set it up in our environment-setup scripts. `./scripts/native-system-benchmark.sh` as added in #259 might offer some inspiration for how to optionally support it.

Reply:
This is an oversight on my end, should not be included here -- good catch